roles/addons/nhc/vars/RedHat.yml

```diff
@@ -0,0 +1,3 @@
+---
+nhc_packages_to_install:
+  - lbnl-nhc
```

Comment:
Where can one find this package? If not www.bluebanquise.com, can you add the information to the readme?

Reply:
Yes, good catch. I will add that in the readme.